### ISSUE TITLE
Unable to find Sophie Locke as a project manager on Supabase!

### ISSUE THREAD

**Issue by sunehajagan** [Blue-Marine-Foundation/Maerl#119]
Can't find Soph on the project_manager_id section (which references public_users) - need to change her as project manager to some projects on Mearl

**Comment by samjamead** [comment]
Hi @sunehajagan yes you're right – it looks like there's no entry for Soph (probably because we haven't needed to list her as a PM before) 

You can create a new entry in the `users` database table for Sophie and then she'll appear

**Comment by samjamead** [comment]
https://supabase.com/dashboard/project/vqfxvjlhtmhqtwcpwddk/editor/28560?schema=public&sort=first_name:asc

<img width="1223" alt="Screenshot 2024-11-19 at 20 11 04" src="https://github.com/user-attachments/assets/a8bd71c7-6ecf-4fcc-9ea0-56fe1c3fcb47">

**Comment by sunehajagan** [comment]
Hi @samjamead - I've added Soph in as a new user, but when I try adding her to the users table on Supabase, I get an error response, please see the screenshot below : 

![Image](https://github.com/user-attachments/assets/6fbf5932-2679-4768-9216-d4f9f6435f7b)

**Comment by samjamead** [comment]
Hey @sunehajagan, thanks for sharing! OK great, that gave me a good clue to go on. And I found something very strange! Sophie's name in the `users` table had been changed to "James Morris" – so an entry did exist for her user id (hence the error you got), but we couldn't find her by searching! 

<img width="583" alt="Screenshot 2024-11-25 at 21 48 58" src="https://github.com/user-attachments/assets/16ba4df2-3e40-4bfa-b899-e7ac67852400">

I have resolved the error now and Sophie is appearing in the projects table in Maerl.
